### ISSUE TITLE
Update Svelte peer dependency range to accommodate Svelte 5

### ISSUE THREAD

**Issue by armchair-traveller** [highcharts/highcharts-svelte#3]
### Current Behavior
The package currently specifies a Svelte peer dependency range of `">=3.0.0 <5.0.0"`, which causes npm warnings when used with Svelte 5.x:

```
├─┬ @highcharts/svelte 1.1.0
│ └── ✕ unmet peer svelte@">=3.0.0 <5.0.0": found 5.1.0
```

### Proposed Change
Update the peer dependency range in `package.json` to include Svelte 5.x:
```json
{
  "peerDependencies": {
    "svelte": ">=3.0.0 <6.0.0"
  }
}
```

### Rationale
- Svelte 5 is now released and being adopted by the community
- Svelte 5 is backwards compatible with Svelte 4 components except in rare cases, usually not the Svelte component's code, but side effects.
  - The current implementation appears compatible with Svelte 5. I have not observed any issues when using `@highcharts/svelte` with Svelte 5.
- This change will eliminate npm warning messages for users on Svelte 5

### Additional Notes
Please let me know if there is any info that would prevent this peer version bump.

**Comment by jszuminski** [comment]
@armchair-traveller thanks for reporting! You're totally right, Svelte 5 is backwards compatible so this package should support it. I've updated the package.json and published v1.1.1. Please test it out and let me know if it works as supposed now.

**Comment by armchair-traveller** [comment]
Thanks for fixing this quickly! Yup, tried it no more peer dep warnings with 5.
